groupcache_test.go

```diff
@@ -285,6 +296,17 @@ func TestCaching(t *testing.T) {
 	if fills != 1 {
 		t.Errorf("expected 1 cache fill; got %d", fills)
 	}
+
+	meta = countMeta(func() {
+		for i := 0; i < 10; i++ {
+			if _, err := stringGroup.Contain(dummyCtx, "TestCaching-key1"); err != nil {
+				t.Fatal(err)
+			}
+		}
+	})
+	if meta != 0 {
```

Comment:
why's this second one supposed to get 0?

Reply:
The gets filled the cache, so the next Contain calls do not need to call the parent.
I'll add some comments.

Reply:
got it - thanks!